Logger/BaseLoggerMixins.cs

```diff
@@ -1,6 +1,54 @@
-﻿namespace Logger;
+﻿using System;
+
+namespace Logger;
 
 public static class BaseLoggerMixins
 {
-    
+    public static void Error(this BaseLogger b, string message, params object[] arguments)
+    {
+        if (b is null)
+        {
+            throw new ArgumentNullException();
```

Comment:
```suggestion
            throw new ArgumentNullException(nameof(logger));
```
Make sure to specify what argument is throwing the null exception